src/lib.rs

```diff
@@ -43,6 +43,10 @@ pub mod poll;
 
 pub mod net;
 
+#[cfg(any(target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd",
```

Comment:
The `linux` is out of alphabetical order, and could you list these 1-per-line? When `cfg`'s get longer than a line we like to wrap with each target on its own line.